.travis.yml

```diff
@@ -0,0 +1,25 @@
+sudo: false
+language: node_js
+cache:
+  directories:
+    - node_modules
+node_js:
+  - '6'
+before_install:
+  - npm i -g npm@^3.0.0
+  - npm i -g surge
+  - npm i -g @schibstedspain/sui-studio
+  - npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN
+after_success:
+  - git remote set-url origin https://${GH_TOKEN}@github.com/SUI-Components/sui-components.git
```

Comment:
I understand you need the token because `suistudio release` creates some type of tag or commit?